test/widgets/emoji_reaction_test.dart

```diff
@@ -556,6 +556,32 @@ void main() {
         debugNetworkImageHttpClientProvider = null;
       });
 
+      testWidgets('text emoji list items must have correct height', (tester) async {
+        // Regression test for #1587
+        tester.platformDispatcher.textScaleFactorTestValue = 2.0;
+        addTearDown(tester.platformDispatcher.clearTextScaleFactorTestValue);
```

Comment:
The reproduction recipe for #1587 doesn't involve setting the system text-size setting, so I don't think these lines are needed in a regression test for #1587. What happens if we remove them?